app/jobs/send_task_delete_notification_job.rb

```diff
@@ -0,0 +1,12 @@
+class SendTaskDeleteNotificationJob < ApplicationJob
+  sidekiq_options queue: :mailers
+  sidekiq_throttle_as :mailer
+  sidekiq_options lock: :until_and_while_executing, on_conflict: { client: :log, server: :reject }
+
+  def perform(task_id)
+    task = Task.find_by(id: task_id)
```

Comment:
Will the task be found after you have destroyed it?

Reply:
No, it will not
Good catch
I can pass some minimal information in params or only disactivate element instead of deleting, right?

Reply:
Sure, you can save the attributes you need before destroying the task and pass them to the job.

Reply:
> Sure, you can save the attributes you need before destroying the task and pass them to the job.

Okay, i fixed it a little